backend/core/database.py

```diff
@@ -509,6 +518,34 @@ def config_request_rpc(self):
                 while self.rules is None:
                     self.connection.drain_events()
 
+        def signal_loading(self, status=False):
```

Comment:
Do you think it would make sense to move the function to utility?

Reply:
Good idea, I will move this.

Reply:
Done, resolving.